src/ks-app/package.json

```diff
@@ -1,58 +0,0 @@
-{
-  "name": "ks-app",
-  "version": "0.0.0",
-  "license": "MIT",
-  "scripts": {
-    "ng": "ng",
-    "start": "ng serve --preserve-symlinks --host 0.0.0.0",
-    "build": "ng build --prod --preserve-symlinks",
-    "test": "ng test",
-    "lint": "ng lint",
-    "e2e": "ng e2e"
-  },
-  "private": true,
-  "dependencies": {
-    "@angular/animations": "^5.0.0",
-    "@angular/common": "^5.0.0",
-    "@angular/compiler": "^5.0.0",
-    "@angular/core": "^5.0.0",
-    "@angular/forms": "^5.0.0",
-    "@angular/http": "^5.0.0",
-    "@angular/platform-browser": "^5.0.0",
-    "@angular/platform-browser-dynamic": "^5.0.0",
-    "@angular/router": "^5.0.0",
-    "@clr/angular": "file:../../dist/clr-angular/clr-angular-0.11.14.tgz",
```

Comment:
I'm pretty sure this was a moved file but I'm having a hard time finding that on github.
We mentioned this on Friday but I want to make sure this is now handled by `angular/cli` right? We should be able to point it directly to the source files now, right?

Reply:
Right, `@clr/angular` is now found by looking at the tslint `path` config for the ks-app. All of our projects (dev, ks-app, clr-angular, clr-icons) except schematics use the same node_modules and package.json at the root through the CLI (want to fix that for schematics sometime as well)